compiler/test/runnable/test23889.c

```diff
@@ -1,8 +1,8 @@
 // DISABLED: win freebsd openbsd
 
-// https://issues.dlang.org/show_bug.cgi?id=23886
+// https://issues.dlang.org/show_bug.cgi?id=23889
 
-#include <stdlib.h>
+#include <alloca.h>
```

Comment:
Were there any regressions wrt. being able to include stdlib.h?

Reply:
yes, both commits before `-std=c11` were meant to fix issues that I encountered when turning it on.

Reply:
Ah sorry, read the commit msg just now. - Okay, and `stdlib.h` is tested by `compilable/stdcheaders.c` already (except Windows), all good.

Reply:
macos man pages say that `alloca` may be used after including either `stdlib.h` or `alloca.h` but on my linux box the man page says only lists `alloca.h`: [macos man](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/alloca.3.html) [linux man](https://man7.org/linux/man-pages/man3/alloca.3.html)

```
[happy@happypc] /tmp % cat a.c
#include <stdlib.h>
int main () {
	alloca(12);
}
[happy@happypc] /tmp % gcc -std=gnu11 a.c
[happy@happypc] /tmp % gcc -std=c11 a.c
a.c: In function ‘main’:
a.c:3:9: error: implicit declaration of function ‘alloca’; did you mean ‘calloc’? [-Wimplicit-function-declaration]
    3 |         alloca(12);
      |         ^~~~~~
      |         calloc
```